libvast/src/pipeline.cpp

```diff
@@ -40,12 +41,12 @@ class local_control_plane final : public operator_control_plane {
     die("not implemented");
   }
 
-  auto schemas() const noexcept -> const std::vector<type>& override {
-    die("not implemented");
+  auto schemas() noexcept -> const std::vector<type>& override {
+    return vast::modules::schemas();
   }
 
-  auto concepts() const noexcept -> const concepts_map& override {
-    die("not implemented");
+  auto concepts() noexcept -> const concepts_map& override {
+    return vast::modules::concepts();
   }
```

Comment:
As far as I can tell these can still be marked `const`, as it doesn't access any state.

Reply:
I reinstated the east-`const`ness for those methods, but for some reason GitHub doesn't show the updated commits on my side.

Reply:
Technical side note:

East or west const refer to whether you want write `const auto` or `auto const` when qualifying a type. The `const` I asked you to add back is not a _const type qualifier_, but rather a _const member function qualfiier_.